cli/zoo.py

```diff
@@ -76,5 +76,55 @@ def manager(port):
     zoo_manager.serve(port)
 
 
+@zoo_cli.command(
+    name="install",
+    help="Attempt to install the specified agents from the given paths/url",
+)
+@click.argument(
+    "agent_paths",
+    type=click.Path(exists=True),
+    metavar="<script>",
+    nargs=-1,
+    required=True,
+)
+def install_agents(agent_paths):
+    if not agent_paths:
+        # nargs=-1 in combination with a default value is not supported
+        # if agent_paths is not given, set the known two zoo agent paths as default
+        agent_paths = ["zoo/policies/open-agent", "zoo/policies/rl-agent"]
+
+    pip_install_cmd = [
+        "pip",
+        "install",
+        ".",
+    ]
+
+    def clean(exec_dir):
+        subprocess.check_call(
+            [sys.executable, "setup.py", "clean", "--all"],
+            cwd=exec_dir,
+        )
+
+    for agent_path in agent_paths:
+        goal_dir = os.path.join(os.getcwd(), agent_path)
+        clean(goal_dir)
+        proc = subprocess.Popen(
+            pip_install_cmd,
+            stderr=subprocess.PIPE,
+            cwd=goal_dir,
+        )
+        proc.wait()
+        stdout, std_err = proc.communicate()
```

Comment:
Is `stdout` used?

Reply:
No, but you need a variable there since `proc.communicate()` returns a tuple.

Reply:
For unused variables, the following will also work: 
```
import subprocess

proc = subprocess.Popen(['echo', 'Hello'], stdout=subprocess.PIPE)
stdout = proc.communicate()[0]
print(stdout)

proc = subprocess.Popen(['echo', 'Hello'], stdout=subprocess.PIPE)
stdout, _ = proc.communicate()
print(stdout)
```